src/main/features/response/draft/responseDraft.ts

```diff
@@ -125,6 +125,10 @@ export class ResponseDraft extends DraftDocument {
       && this.howMuchPaidClaimant.option === HowMuchPaidClaimantOption.AMOUNT_CLAIMED
   }
 
+  public isDigitalResponseSelected (): boolean {
```

Comment:
@varma02 implementation is not right + it is difficult to read.

You would probably understand it yourself if you put some unit tests.